prowler/lib/outputs/models.py

```diff
@@ -85,6 +85,18 @@ def generate_provider_output_csv(
             )
             finding_output = output_model(**data)
 
+        if provider == "kubernetes":
+            data["resource_id"] = finding.resource_id
+            data["resource_name"] = finding.resource_name
+            data["namespace"] = finding.namespace
+            data[
+                "finding_unique_id"
+            ] = f"prowler-{provider}-{finding.check_metadata.CheckID}-{finding.namespace}-{finding.resource_name}"
```

Comment:
Why not to use the `finding.resource_id` as the last part of the unique id?

Reply:
Good idea, I will change it